src/olympia/shelves/migrations/0002_auto_20200716_1254.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 2.2.14 on 2020-07-16 12:54
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('shelves', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='shelf',
+            old_name='shelfType',
+            new_name='shelf_type',
+        ),
+        migrations.RemoveField(
+            model_name='shelf',
+            name='footerPathname',
+        ),
```

Comment:
It's not worth doing here because it's all being landed at once with no existing data but this migration would need manually fixing if 0001 was already deployed.

Django hasn't picked up that you've renamed the field and altered it at the same time so it thinks you've deleted the field and then added a new one.  To fix you'd change the AddField to an AlterField and then move and replace RemoveField with RenameField after it

Reply:
Makes sense, I'll be sure to check the migrations moving forward!